JAWS

adajaws opened the pull request:
Updating the **JAWS** pool with **correct** information provided in this ticker, since registration fee did not pull through for the first time, and needed to start from scratch (now it's ok).


Please void **VOID98** and remove **VOID98** registration data of PR #1764 from the main branch.

sdalmeida commented:
Hi @adajaws
Looks like your ticker is already taken. In order to have the old one removed, you first need to rename the original pool to VOID.
Once it has been merged, then the ticker JAWS will be available again.

adajaws commented:
@sdalmeida Do I need to update the also the old .json and sign the sig file, or just update the ticker name?

adajaws commented:
@sdalmeida I have renamed the old ticker, but could not update the old .json and sig files into main branch (so the pool name would be as VOID99 in the .json file). Is this now ok?

adajaws commented:
Now I have done the renaming, now it should be ok?

adajaws commented:
Trying again.

adajaws commented:
-

adajaws commented:
Hi.

Thank you for your feedback but it did not help us to go all The way to the finish line as we would need to remove the incorrect JAWS pool data and replace the data with the correct one.

We have now created the commit into github as instructed as follows:

1) We have now close the request #1838 on order to start from beginning.

2) We have renamed the old merged tickername #1764 from github from JAWS to VOID99

3)  We have updates the old pool .json and .sig files as signed with the old pool owners secret key and updated the pool name in .json file from JAWS into VOID.

4) We have committed the updated .json and .sig files into new pull request using git commit -m "VOID99" and created pull request which opened the ticker named JAWS for number  #1848.

5) We have commented into the #1764 that the merged ticker (VOID99) with pool information needs to be removed

6) we have updates comments to the ticker #1848 to remove VOID99 ticker data from main branch

The automatic check goes now through with no errors.

7) We tried to update the the new created .json and .sig -files by copying them Into my local repository on Linux and use git add pknnnnn*.json and .sig then commit -m "JAWS" and finally git push but the new files appeared under the same ticker named JAWS with #1848 and then the buildkite script shows on red with an error that commit should Be exactly two files not four (old json and sig + new json and sig). So we removed the commit for adding also the updated files for #1848 ticker named as JAWS, since the buildkite does not go through.

8) Do we need to change the name of the new pull request #1848 also to VOID99? Or what we should do next? Or do we need to wait for merge and then create new pull request?

9) Could you instruct ja how to proceed to the finnish since we really want that nobody else is stealing our jaws pool name and need to remove the old pool information since it was containing wrong information since the pool registration fee did not to through because too little adas.

Now we have new pool with sk/pk and address so we cannot use the old address and old ok in the json file, since the registration fee has been paid with the new address and now it shows on the blockchain.

We would really appreciate, If somebody could help us to run to the finish line. We have been gone through all Cardano forums and tried to find help from telegram groups with no real help.

I understand that you have lot of people asking you advices, but we would not ask for your help If we could find the solution by our own.

Thanking you for helping us to go to stable phase and best regards, Ada

________________________________
From: Simon <notifications@github.com>
Sent: Monday, February 3, 2020 3:56:13 AM
To: cardano-foundation/incentivized-testnet-stakepool-registry <incentivized-testnet-stakepool-registry@noreply.github.com>
Cc: ada jaws <adajaws.pool@gmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [cardano-foundation/incentivized-testnet-stakepool-registry] JAWS (#1837)


Hi @adajaws<https://github.com/adajaws>
Looks like your ticker is already taken. In order to have the old one removed, you first need to rename the original pool to VOID.
Once it has been merged, then the ticker JAWS will be available again.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/cardano-foundation/incentivized-testnet-stakepool-registry/pull/1837?email_source=notifications&email_token=AOMSNFJ7IWWGMN3MC2OMRC3RA52T3A5CNFSM4KO4HHBKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEKSIFZY#issuecomment-581206759>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AOMSNFNVWMX2T3HOUKKXOG3RA52T3ANCNFSM4KO4HHBA>.
